docs/graphcache/errors.md

```diff
@@ -9,7 +14,7 @@ message may not actually tell you everything about what's going on.
 This is a supporting document that explains every error and attempts to give more
 information on how you may be able to fix some issues or avoid these errors/warnings.
 
-## (1) Invalid GraphQL document <a id="1"></a>
+## (1) Invalid GraphQL document
```

Comment:
The anchor tags could potentially also just go above the headings and we can see if that works

Reply:
Good shout, let me check